### DISCUSSION TITLE
Why does turborepo use ".eslintrc" and not "eslint.config.js"?

### DISCUSSION THREAD

**Discussion by yikl1004** [vercel/turborepo#8533] [Help]
### Summary

Was this a choice considering existing plugins or expandability?
I'm curious as to why you chose to use a ".eslintrc" file.

What do you guys think?
Does anyone know the exact reason?

**Comment by anthonyshew** [comment]
Turborepo doesn't care which file type you use. Are you meaning in the examples?
